imageio/v2.py

```diff
@@ -268,6 +268,8 @@ def get_reader(uri, format=None, mode="?", **kwargs):
         Used to give the reader a hint on what the user expects (default "?"):
         "i" for an image, "I" for multiple images, "v" for a volume,
         "V" for multiple volumes, "?" for don't care.
+    pilmode : str
+        Used to provide mode parameter that need to be passed to the reader
```

Comment:
```suggestion
```

Correspondingly with my previous comment, we will not need to document `pilmode` here if we move it to `LegacyReader`.